qless/workers/__init__.py

```diff
@@ -46,10 +47,10 @@ def title(cls, message=None):
     @classmethod
     def divide(cls, jobs, count):
         '''Divide up the provided jobs into count evenly-sized groups'''
-        jobs = list(zip(*itertools.izip_longest(*[iter(jobs)] * count)))
+        jobs = list(zip(*_compat.izip_longest(*[iter(jobs)] * count)))
         # If we had no jobs to resume, then we get an empty list
         jobs = jobs or [()] * count
-        for index in xrange(count):
+        for index in range(count):
```

Comment:
or use while index < count and index += 1 in the loop. The compat shim was small enough that I didn't feel like adding a dependency on six.